types/nuxt-i18n.d.ts

```diff
@@ -26,6 +26,7 @@ declare namespace NuxtVueI18n {
 
     interface DetectBrowserLanguageInterface {
       useCookie?: boolean
+      cookieDomain?: string
```

Comment:
`string | null` since with strict null checking the `?` means undefined so it wouldn't match default value.